### PR TITLE
Add docs for env config

### PR DESCRIPTION
**Pull request by Chemaclass**
## 🔖 Changes

- Add a new documentation page for env-vars

## 🖼️ Demo

https://github.com/TypedDevs/bashunit/assets/5256287/10049843-dffe-4322-9f5c-ef03d64fe4d2

## ✅ To-do list

- [x] I updated the `CHANGELOG.md` to reflect the new feature or fix
- [x] I updated the documentation to reflect the changes

### CODE REVIEW

**Review by Tito-Kati — CHANGES_REQUESTED** [review]
Could you put the defaults values too and change the texts to indicate what happens in case of being true? In the ones you refer to the command line, it would be nice to add what they do so you don't have to be jumping from one side to the other while reading the documentation.

And I think I would call the section just "Configuration".

The definition of the section could be a tad more commercial and explain a bit about its purpose, not so robotic, you could for example say that it serves to configure the behavior of bashunit in your project, that they need to create a .env file in the root directory but they can give it another name if they pass it as an argument to the command with --env (and a link to the corresponding section with more detail). I would also remove the example env or if you include it, provide one with values and not all arguments so that people can use it as an example.

In summary, if I read that documentation without any context currently, I wouldn't know how to configure bashunit.

**Comment by Tito-Kati** [comment]
And I also think it's not necessary to put the bashunit command in every example, with the .env and the Output it's enough.

**Comment by Tito-Kati** [comment]
Ah, and remember to update the command line section to say that they override the environment configuration and also add a link to the environment configuration if they wanted to make the change permanent.

**Review by Tito-Kati — APPROVED** [review]
